README.md

```diff
@@ -27,19 +63,24 @@ coordinates will have a third component containing elevation added.
 
 You can check the health of the server with the `/status` endpoint.
 
-
 ## Testing
 
-````bash
-npm install
-npm run test
-````
+Tests are written using Node's native test runner, and are intended to run against production (i.e.,
+TSC-compiled) code. They use the `.env` environment variable, and also require AWS credentials
+(loaded from `~/.aws`).
```

Comment:
Not sure the lift but could we mock the AWS calls to not require AWS creds for the test?

Reply:
I initially tried to mock the `s3Fetcher` but ran into technical problems. I can bore you with the details.

These tests are currently only run manually, and I think they're more like e2e tests than unit tests, so I think it's appropriate for them to use the "real thing" and not a mock.

I don't anticipate the tests running dozens or more times a day, like GaiaCloud tests.